modules/ROOT/pages/to-request-access.adoc

```diff
@@ -69,7 +69,11 @@ The OAuth 2 grant type:
 
 == API Groups
```

Comment:
GLOBALLY attempt to reword topic titles so that they are more precise to the actual content, and not vague:
```suggestion
== API Group Access
```
OR
```suggestion
== Access to API Groups
```
OR
```suggestion
== Types of API Group Access
```